bootstrap.sh

```diff
@@ -6,8 +6,8 @@ HOSTBIN=$(go env GOHOSTOS)_$(go env GOHOSTARCH)/bin
 export GOBIN=$(pwd)/$HOSTBIN
 echo GOBIN is now $GOBIN
 
-echo Building the build tool into $HOSTBIN
-GO111MODULE=on go get ./util/src/harvey/cmd/...
+echo Building harvey-os.org commands into $HOSTBIN
+GO111MODULE=on go get harvey-os.org/cmd/...
 
 echo Building u-root into $HOSTBIN
 GO111MODULE=on go get github.com/u-root/u-root@c370a343c8b0b01faac358c1dafb409e5576ae1a
```

Comment:
Maybe add `github.com/u-root/u-root` to `tools.go` also, so that you don't need to provide the hash.

Reply:
i tried that, but because of the u-root tags not playing nicely with go modules, I couldn't get it to work.